frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts

```diff
@@ -105,63 +84,177 @@ if (clusterStatus && cephHealth) {
           `oc -n ${NS} rsh  $(oc -n ${NS} get pod | grep ceph-operator| awk '{print$1}') ceph --conf=/var/lib/rook/${NS}/${NS}.config osd tree --format=json`,
         ).toString(),
       );
-
       expansionObjects.previousOSDIds = getIds(expansionObjects.previousOSDTree.nodes, OSD);
 
-      await goToInstalledOperators();
-      await click(ocsOp);
-      const storageClusterLink = await currentSelectors.getStorageClusterLink();
-      await click(storageClusterLink);
+      await selectSCDropdown(expansionObjects.uid);
 
-      await clickKebabAction(uid, 'Add Capacity');
-      await verifyFields();
+      // eslint-disable-next-line no-useless-escape
+      expansionObjects.defaultSC = execSync(
+        `kubectl get storageclasses | grep -Po '\\w+(?=.*default)'`,
+      )
+        .toString()
+        .trim();
+      await click(getSCOption(expansionObjects.defaultSC));
```

Comment:
Why the default SC we should keep this configurable based on the environment variable?

Reply:
Because default sc will always be present.